Makefile

```diff
@@ -110,7 +111,19 @@ LINT_SRC := $(filter-out %_test.go ./.gen/%, $(ALL_SRC))
 # downloads and builds a go-gettable tool, versioned by go.mod, and installs
 # it into the build folder, named the same as the last portion of the URL.
 define go_build_tool
-@echo "building $(or $(2), $(notdir $(1))) from $(1)..."
+@echo "building $(or $(2), $(notdir $(1))) from internal/tools/go.mod..."
+@go build -mod=readonly -modfile=internal/tools/go.mod -o $(BIN)/$(or $(2), $(notdir $(1))) $(1)
```

Comment:
won't this fail your `@` rule?

Reply:
if merged without updating it, yes.
stacked PRs are not a thing github supports though, so I have to either include its changes in this PR, or wait for it to merge first.

Reply:
updated!